hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieFileGroupReaderBasedRecordReader.java

```diff
@@ -101,8 +102,7 @@ org.apache.hadoop.mapred.RecordReader<NullWritable, ArrayWritable> getRecordRead
 
   public HoodieFileGroupReaderBasedRecordReader(HiveReaderCreator readerCreator,
                                                 final InputSplit split,
-                                                final JobConf jobConf,
-                                                final Reporter reporter) throws IOException {
```

Comment:
reporting progress. I changed it so that the arg gets passed directly when creating the readerCreator. It was only being used by the reader creator, so this simplifies the code by not needing to pass an extra variable around